src/components/application_manager/rpc_plugins/vehicle_info_plugin/src/vehicle_info_pending_resumption_handler.cc

```diff
@@ -153,6 +157,7 @@ void VehicleInfoPendingResumptionHandler::SendHMIRequestForNotSubscribed(
   const auto remaining_subscriptions = pending_resumption.NotSubscribedData();
   auto request = CreateSubscribeRequestToHMI(remaining_subscriptions);
   const auto corr_id = get_corr_id_from_message(*request);
+  corr_id_ = corr_id;
```

Comment:
@ychernysheva you may remove extra variable here:
```suggestion
  corr_id_ = get_corr_id_from_message(*request);
```